Added info about the values of the HTTP_PROXY and HTTPS_PROXY var (#1883)
MicrosoftDocs/azure-docs-powershell: docs-conceptual/azps-6.3.0/az-powershell-proxy.md

On systems where environment variables are case-sensitive, the variable names may be all lowercase
or all uppercase. The lowercase names are checked first.

> The proxy server may be a hostname or IP address, optionally followed by a colon and port number, or
> it may be an http URL, optionally including a username and password for proxy authentication. The
> URL must be start with http, not https, and cannot include any text after the hostname, IP, or port.

Reviewer: ```suggestion
The proxy server may be a hostname, IP address, or HTTP URL, optionally followed by a colon and port number. Username and password are also optional for proxy authentication. The
URL must start with HTTP, not HTTPS, and cannot include any text after the hostname, IP address, or port number.
```

Author: I do not know if the environment variable is case sensitive so I would prefer using `http` over HTTP.
